pyvista/core/filters/poly_data.py

```diff
@@ -2088,8 +2088,8 @@ def flip_normals(poly_data):
         >>> sphere.plot_normals(mag=0.1, opacity=0.5)
 
         """
-        if not poly_data.is_all_triangles:
-            raise NotAllTrianglesError('Can only flip normals on an all triangle mesh')
+        if not poly_data.is_all_triangles():
```

Comment:
This is probably because people expect that `is_all_triangles` is a property since it sounds like a noun.

We might be a little late for a refactor since calling a property that's evaluated to a bool won't make sense.

Reply:
We could possibly deprecate this by switching to a property and returning a bool that has been subclassed to be callable.

Reply:
Another PR for that then.